gcp/iam/workload_identity_federation/vars.tf

```diff
@@ -0,0 +1,27 @@
+variable project_id {
+  type = string
+}
+
+variable workload_identity_pool {
+  type = object({
```

Comment:
For this particular repo, I'm not really sure why we made this an object? Why can't each of these be variables on their own?

Reply:
Updated this to be their own variables